packages/block-library/src/post-comments-link/block.json

```diff
@@ -39,6 +39,9 @@
 			"__experimentalDefaultControls": {
 				"fontSize": true
 			}
+		},
+		"interactivity": {
+			"clientNavigation": true
```

Comment:
I'm not sure if this one has JS. Can you check?

Reply:
Checked, it seems it has not. It was `comments` and `post-comments-form`. I won't include `comment-reply` too just in case.
![Screenshot 2024-02-02 at 10 58 39](https://github.com/WordPress/gutenberg/assets/37012961/b4b26d52-039c-4855-aa27-63e0ddada9c9)

Reply:
Ok, thanks.